bigquery_etl/shredder/amplitude.py

```diff
@@ -132,7 +144,7 @@ def _deletions(start):
                 "user_ids": ids[start:end],
                 "requester": "shredder",
                 "ignore_invalid_id": "True",
-                "delete_from_org": "True",
+                "delete_from_org": "False",
```

Comment:
provided credentials should be for the relevant project, so we don't want to `delete_from_org`

Reply:
breaking this bit out into a separate PR